src/components/00-design-tokens/_colors.scss

```diff
@@ -0,0 +1,34 @@
+$c-white: #fff;
```

Comment:
@aekong Do you feel like these need to be namespaced? I'm not sure we'll ever be in danger of having two conflicting `$white`s or `$blue-1`s, but maybe you feel differently